docs/UserGuide.md

```diff
@@ -60,7 +104,7 @@ in this user guide.
 
 <br>
 
-## Quick start
+## **Quick start**
```

Comment:
For the example in step 4 (lines 133 - 134), could you help to italicise the sentence and bold **For example,**, to keep it consistent with the other examples in the user guide?